python/tutorial1_display.py

```diff
@@ -92,18 +66,17 @@
         output_byte_size *= 3
 
     loop_num = 100
+    coef = pow(2, num_bit_shift)
 
     for x in range(loop_num):
-        port_map.set_u1(dispose_p, x==loop_num-1)
-
         # running the builder
-        builder.run(port_map)
+        builder.run()
         output_bytes = outputs[0].read(output_byte_size) 
```

Comment:
`read` is no longer needed. You can create ion.Buffer passing numpy array  (`Buffer(array=numpy_array)`) and you can access data through numpy array without `ion.Buffer.read/write` operation

Reply:
fixed in 24b9ed0